### ISSUE TITLE
from twitterscraper import query_tweets

### ISSUE THREAD

**Issue by Abimbola2505** [taspinar/twitterscraper#359]
I am trying to run the code from twitterscraper import query_tweets from Jupyter through acanonda but having this error:

ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

Please help.

**Comment by DSqiansun** [comment]
i have the same issue

**Comment by olalekan-bodunde** [comment]
i have the same issue as well. anyone help please

**Comment by Altimis** [comment]
Twitterscrapper doesnt seem to be working anymore, it seems like Twitter banned it. You can use [Scweet](https://github.com/Altimis/Scweet) instead.

**Comment by MakkiNeutron** [comment]
You are right Altimis, i tried everything but in colab and even anaconda i could not find solution of this 'cannot import name 'query_tweets' from partially initialized module 'twitterscraper' '
